streams/ge_itdc.m3u

```diff
@@ -0,0 +1,113 @@
+#EXTINF:-1 tvg-id="",Amedia 2
+https://tbs01-edge11.Itdc.ge/amedia2/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Amedia Premium
+https://tbs01-edge11.Itdc.ge/amediapremium/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Киносерия
+https://tbs01-edge11.Itdc.ge/kinoseria/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Индия
+https://tbs01-edge11.Itdc.ge/zeetv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",24 Техно
+https://tbs01-edge11.Itdc.ge/24techno/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Авто плюс
+https://tbs01-edge11.Itdc.ge/autoplus/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Авто 24
+https://tbs01-edge11.Itdc.ge/auto24/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",365 Дней
+https://tbs01-edge11.Itdc.ge/365day/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Оружие
+https://tbs01-edge11.Itdc.ge/oruzhie/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Зоопарк
+https://tbs01-edge11.Itdc.ge/zoo/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Кто есть кто
+https://tbs01-edge11.Itdc.ge/ktoestkto/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Кухня тв
+https://tbs01-edge11.Itdc.ge/kuxnia/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Домашний
+https://tbs01-edge11.Itdc.ge/domashni/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Пятница
+https://tbs01-edge11.Itdc.ge/friday/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Перец
+https://tbs01-edge11.Itdc.ge/perec/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Стс
+https://tbs01-edge11.Itdc.ge/ctctv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Твц
+https://tbs01-edge11.Itdc.ge/tvc/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Мир 24
+https://tbs01-edge11.Itdc.ge/mir24/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Мир
+https://tbs01-edge11.Itdc.ge/mir/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Тнт music
+https://tbs01-edge11.Itdc.ge/rutv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Abkhaz
+https://tbs01-edge11.Itdc.ge/abkhaz/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Egrisi tv
+https://tbs01-edge11.Itdc.ge/egrisitv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Nwbctv
+https://tbs01-edge11.Itdc.ge/nwbctv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Stereoplus
+https://tbs01-edge11.Itdc.ge/stereoplus/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Music box
+https://tbs01-edge11.Itdc.ge/musicbox/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Music tv
+https://tbs01-edge11.Itdc.ge/musictv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Ностальгия
+https://tbs01-edge11.Itdc.ge/nostalgy/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Da vinci
+https://tbs01-edge11.Itdc.ge/davinci/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Детский мир
+https://tbs01-edge11.Itdc.ge/detskymir/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Уникум
+https://tbs01-edge11.Itdc.ge/detsky/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Viasat sport
+https://tbs01-edge11.Itdc.ge/viasatsport/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Mma-tv.com
+https://tbs01-edge11.Itdc.ge/m1/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",HD Life
+https://tbs01-edge11.itdc.ge/hdlife/tracks-v1a1/mono.m3u8
```

Comment:
Duplicate.

Reply:
haven't found any duplicate for m1. Fixed